modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -484,11 +485,7 @@ object SeqexecEngine {
 
   // scalastyle:off
   def seqexecConfiguration(giapi: Giapi[IO]): Kleisli[IO, Config, Settings] = Kleisli { cfg: Config =>
-    val site = cfg.require[String]("seqexec-engine.site") match {
-      case "GS" => Site.GS
-      case "GN" => Site.GN
-      case _    => Site.GS // Let's default to GS
-    }
+    val site                    = Site.fromTag(cfg.require[String]("seqexec-engine.site")).getOrElse(Site.GS)
```

Comment:
Can you define a typeclass instance here so you can say `cfg.require[Site]` rather than inlining the logic here? The benefit is that you explicitly say that the `tag` value is how we encode sites in the config.

Reply:
That's a good idea, let's see if I can figure it out